test/scripts/attest_fn_call.txtar

```diff
@@ -1,10 +1,8 @@
 # [execute] call the bky-as CLI to attest a function call
 stdin fn-call.json
 exec bky-as attest-fn-call
-cp stdout result.json
+cp stdout out.json
```

Comment:
@tahardi I talked with Dave about the goal of testing. He actually took a different view in that this testing should use the example code in this repo, but the testing flow should reflect as closely as possible the flow that the documentation tells the users to follow. That makes sense. Engineering can keep the code up to date, but product keeps the documentation and these tests up to date. Engineering could add other tests if necessary, but maybe that's down the line.

So, in that vein, I'm updating the naming here to reflect what's in the documentation. Note that I'm also updating the documentation to avoid telling users to use `make` (other than `make build`, which until we switch to `bky-c` is still the best option).